packages/blockchain/algo/src/lib/services/algo.tx.ts

```diff
@@ -552,6 +659,126 @@ export const algoTxService = (args: { algoWeb: AlgoWeb }, apiCalls: AlgoApiCalls
+        /**
+         * Sign Algorand create Fractional NFT transaction with private key locally. Nothing is broadcast to the blockchain.
+         * @param body content of the transaction to broadcast
+         * @param testnet mainnet or testnet version
+         * @param provider url of the Algorand Server to connect to. If not set, default public server will be used.
+         * @returns transaction data to be broadcast to blockchain.
+         */
+        createFractionalNFTSignedTransaction: async (
+          body: MintMultiToken | MintMultiTokenKMS,
+          testnet = false,
+          provider?: string,
+        ) => prepareCreateFractionalNFTSignedTransaction(body, testnet, args.algoWeb, provider),
+        /**
+         * Sign Algorand transfer Fractional NFT transaction with private key locally. Nothing is broadcast to the blockchain.
+         * @param body content of the transaction to broadcast
+         * @param testnet mainnet or testnet version
+         * @param provider url of the Algorand Server to connect to. If not set, default public server will be used.
+         * @returns transaction data to be broadcast to blockchain.
+         */
+        transferFractionalNFTSignedTransaction: async (
+          body: TransferMultiToken | TransferMultiTokenKMS,
+          testnet = false,
+          provider?: string,
+        ) => prepareTransferFractionalNFTSignedTransaction(body, testnet, args.algoWeb, provider),
+        /**
+         * Sign Algorand burn Fractional NFT transaction with private key locally. Nothing is broadcast to the blockchain.
+         * @param body content of the transaction to broadcast
+         * @param testnet mainnet or testnet version
+         * @param provider url of the Algorand Server to connect to. If not set, default public server will be used.
+         * @returns transaction data to be broadcast to blockchain.
+         */
+        burnFractionalNFTSignedTransaction: async (
+          body: BurnMultiToken | BurnMultiTokenKMS,
+          testnet = false,
+          provider?: string,
+        ) => prepareBurnFractionalNFTSignedTransaction(body, testnet, args.algoWeb, provider),
+      },
+      send: {
+        /**
+         * Send signed Algorand create Fractional NFT transaction with private key locally.
+         * This method broadcasts signed transaction to the blockchain. This operation is irreversible.
+         * @param body content of the transaction to broadcast
+         * @param testnet mainnet or testnet version
+         * @param provider url of the Algorand Server to connect to. If not set, default public server will be used.
+         * @returns transaction data to be broadcast to blockchain.
+         */
+        createFractionalNFTSignedTransaction: async (
+          body: MintMultiToken | MintMultiTokenKMS,
+          testnet = false,
+          provider?: string,
+        ) => {
+          const txData = await prepareCreateFractionalNFTSignedTransaction(
```

Comment:
you dont wnna do this. logic should be (as a pseudocode)

`if ('signatureId' in body) {
            return ApiServices.blockchain.algo.algorandBlockchainTransfer(JSON.parse(txData))
          }
          return AlgorandService.algorandBroadcast(await prepareTransferFractionalNFTSignedTransaction(
            body,
            testnet,
            args.algoWeb,
            provider,
          ))`

you must send tx to API, if signatureId is there, dont do any local operations